### PR TITLE
Assignment_2_Swapna

### PR DESCRIPTION
**Pull request by SwapnaWJun20**
New file - To check the steps

### CODE REVIEW

File: src/swapnaWJun20/Demo.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package swapnaWJun20;
+
+public class Demo {
+
```
Comment (KrishnaKTechnocredits) [inline src/swapnaWJun20/Demo.java:4]: Remove space.

File: src/swapnaWJun20/Demo.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package swapnaWJun20;
+
+public class Demo {
+	
+	void Display(){
```
Comment (KrishnaKTechnocredits) [inline src/swapnaWJun20/Demo.java]: Method name should be start with small letter.

File: src/swapnaWJun20/Demo.java (hunk 1 above)
Comment (SwapnaWJun20) [reply src/swapnaWJun20/Demo.java:4]: Done

File: src/swapnaWJun20/Demo.java (hunk 2 above)
Comment (SwapnaWJun20) [reply src/swapnaWJun20/Demo.java]: Done

**Pull request merged**